security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -6,18 +6,35 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The Security Profiles Operator provides a way to define secure computing (https://kubernetes.io/docs/tutorials/security/seccomp/[seccomp]) and SELinux profiles as custom resources, synchronizing profiles to every node in a given namespace.
+The Security Profiles Operator provides a way to define secure computing (http://kubernetes.io/docs/tutorials/security/seccomp/[seccomp]) and SELinux profiles as custom resources, synchronizing profiles to every node in a given namespace.
 
 These release notes track the development of the Security Profiles Operator in {product-title}.
 
 For an overview of the Security Profiles Operator, see xref:../../security/security_profiles_operator/spo-overview.adoc#spo-overview[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-9-0"]
+== Security Profiles Operator 0.9.0
+
+The following advisory is available for the Security Profiles Operator 0.9.0:
+link:http://access.redhat.com/errata/RHBA-2024:XXXX[RHBA-2024:XXXX - OpenShift Security Profiles Operator update]
+
+[id="spo-0-9-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, when using the log enricher to record a SelinuxProfile, the process would fail to create the profile because it was incorrectly identified as a seccomp profile. With this update, the log enricher correctly identifies the profile type and creates the SelinuxProfile as expected. (link:http://issues.redhat.com/browse/OCPBUGS-60139[*OCPBUGS-60139*])
+
+* Previously, when you created a ProfileRecording object, the associated workloads could fail to start and would get stuck in an `Init:CreateContainerError` state. This was caused by an incorrect file path to the seccomp profile. With this update, the file path is corrected, and workloads now start as expected. (link:http://issues.redhat.com/browse/OCPBUGS-60048[*OCPBUGS-60048*])
+
+* Previously, the spod pods could fail to start, entering a CrashLoopBackOff state due to an error parsing semanage configuration file. This issue was caused by an error in the SELinux configuration files. With this update, the configuration files have been corrected, and the spod pods now start successfully. (link:http://issues.redhat.com/browse/OCPBUGS-55829[*OCPBUGS-55829*])
```

Comment:
Better to update the root cause description as below:
"The issue is cused by RHEL 9 image naming convention change in OpenShift 4.19. With this update, the issue has been corrected, and the spod pods now start successfully."